test/serde/msgpack/test_msgpack_serde_full.py

```diff
@@ -50,15 +50,14 @@
 samples[
     syft.frameworks.torch.tensors.interpreters.precision.FixedPrecisionTensor
 ] = make_fixedprecisiontensor
-samples[syft.frameworks.torch.tensors.decorators.logging.LoggingTensor] = make_loggingtensor
```

Comment:
We should probably still test `LoggingTensor`, both because we don't want to lose test coverage and because I think the serdes tests check that all the classes with serialization methods actually get tested.

Reply:
I put it just below with its frameworks friend

Reply:
Ohhhh